src/pages/Accounts/index.tsx

```diff
@@ -64,25 +64,37 @@ const getAccountsColumns = ({ formatAddress, onShowSendFundsModal, onShowWithdra
     width: '25%',
     field: 'accountInfo',
     render(accountInfo: AccountInfo) {
-      const { KSM } = accountInfo.balance || { KSM: accountInfo.deposit || 0 };
+      const KSM = accountInfo.deposit || 0;
       const isDeposit = !!accountInfo.deposit;
-      return (
```

Comment:
Make a component out of it, current state is unreadable. Put "if" inside that extracted component.

Don't see reason to use "else"